source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -3,256 +3,311 @@
-   Long Term Support (LTS) kernel is required for |VB| driver compatibility.
+   .. note::
 
+      In :guilabel:`Advanced options`, choose only **kernel-lts**. The
+      Long Term Support (LTS) kernel is required for |VB| driver compatibility.
 
-#. After |CL| installation is complete, the VM will reboot and return to the 
-   |CL| installer. 
+#. After |CL| installation is complete, the VM will reboot and return to the
+   |CL| installer.
 
-.. note::
-   To release the mouse cursor from the VM console window, press the right Ctrl key on the keyboard.
+   .. note::
 
+      To release the mouse cursor from the VM console window, press the right
+      :kbd:`Ctrl` key on the keyboard.
 
 Unmount the installation ISO
 ============================
 
 The |CL| installer ISO needs to be unmounted to allow the VM to boot from the
-virtual hard disk, which |CL| has been installed to.
-
+virtual hard disk where |CL| has been installed.
 
 #. Power off the |CL| VM.
 
-   .. image:: ./figures/vbox/vbox-shutdown-vm.png
+   .. figure:: figures/vbox/virtualbox-cl-installer-11.png
+      :scale: 100%
       :alt: Powering off a VirtualBox VM
 
-#. Click *Settings* to configure the |CL| VM.
+      Figure 11: Powering off a VirtualBox VM
 
-   .. image:: ./figures/vbox/vbox-vm-created.png
-      :alt: A VM selected in VirtualBox Manager
+#. Click *Settings* to configure the |CL| VM.
 
-#. From the *VM - Settings* window, navigate to the *Storage* pane from the 
+#. From the *VM - Settings* window, navigate to the *Storage* pane from the
    left-hand side.
 
-#. From the middle *Storage Devices* column, click the blue CD disk labeled  
+#. From the middle *Storage Devices* column, click the blue CD disk labeled
    *clear-<VERSION>-installer.iso* under the *Controller: IDE* from.
 
-#. From the right-hand *Attributes* column, click the blue CD disk next to 
-   the *Optical Drive* drop down menu and click *Remove Disk from Virtual 
+#. From the right-hand *Attributes* column, click the blue CD disk next to
+   the *Optical Drive* drop down menu and click *Remove Disk from Virtual
    Drive*
 
-   .. image:: ./figures/vbox/vbox-vm-settings-unmount-ISO.png
-      :alt: Unmounting an ISO in VirtualBox VM Settings        
+   .. figure:: figures/vbox/virtualbox-cl-installer-12.png
+      :scale: 100%
+      :alt: Remove Disk from Virtual Drive
+
+      Figure 12: Remove Disk from Virtual Drive
 
-#. Click *OK* to exit the *VM Settings* menu and return to the main 
+#. Click *OK* to exit the *VM Settings* menu and return to the main
    |VBM|.
 
+.. _vbox-troubleshooting-begin:
 
+Troubleshooting
+***************
 
+#. **Problem:** Out of disk space inside of |CL| and not be able to install
+   additional bundles.
 
-.. include:: virtualbox.rst
-   :start-after: vbox-start-vm-and-lga-begin:
-   :end-before: vbox-start-vm-and-lga-end:
+   **Solution:** The |CL| images are small to minimize download time and
```

Comment:
This shouldn't be applicable in this case because we're installing to a new virtual HD, not running from a live image. Or did I miss an extra section in this guide where we convert the .img to a VDI via vboxmanage?

Reply:
Good catch. Removed. Result of moving Troubleshooting to this doc from previous `virtualbox.rst` which did convert .img to a VDI.